### ISSUE TITLE
[BUG] Sidebar - collapse/expand not working mouseenter/leave

### ISSUE THREAD

**Issue by dtdono0** [ColorlibHQ/AdminLTE#2444]
**Describe the bug**
We have the sidebar menu and push menu button working on our website.  However a user recently reported the mouseover functionality wasn't working to expand the current menu item when hovering over one of the menu's icons in the collapsed state.

I tried to apply a quick fix just to pop the entire menu out:

```
        $(document).ready(function () {
            $('.sidebar').mouseenter(function() {
	            $('[data-toggle="push-menu"]').pushMenu('expand');
            });
            $('.sidebar').mouseleave(function() {
	            $('[data-toggle="push-menu"]').pushMenu('toggle');
            });            
        });
```

But the collapsed menu will only expand when my mouse leaves the highlighted icon region (there's a small few pixel padding to the left of the menu's icons and the edge of the sidebar div).  Although this pops out the entire menu and not just the current item in the menu.

The strange thing is we have the sidebar on another site and the mouseover works as expected.  I'm not sure the best way to debug this issue.  Should I re-download a fresh copy of AdminLTE and see if that works?

**Environment (please complete the following information):**
 - AdminLTE Version: (v2.4.8)
 - Operating System:  Windows 8/10
 - Browser (Version): Chrome (Latest)

**Comment by REJack** [comment]
>  Should I re-download a fresh copy of AdminLTE and see if that works?

That would be best, mostly this bug comes from an not correct HTML markup.

**Comment by dtdono0** [comment]
Sorry I didn't follow up on this sooner.  I took another look at it and noticed a horizontal scrollbar was popping up whenever I hovered over an icon on the side bar.  By scrolling over I noticed the menu was popping out but it is getting constrained to the sidebar area and not just not expanding over the <body> of the page.

I tried adding: data-toggle="tooltip" data-container="body" to my anchor tags, but the hover menu is still not showing over the main content area.  Re-adding the .min.css/js files didn't seem to change anything either.  I'm not sure if this is being caused by AngularJS.

https://i.imgur.com/qMtekFG.png
